docs/06-concepts/18-testing/01-get-started.md

```diff
@@ -170,24 +180,39 @@ tags:
 
 ```
 
+6. Finally, add the `test` and `serverpod_test` packages as dev dependencies in `pubspec.yaml`:
+
+```yaml
+dev_dependencies:
+  serverpod_test: 2.1.4 # Should be same version as the `serverpod` package
```

Comment:
Suggestion: Change this to `<serverpod version>` so that we are compatible with minor versions that this guide is used for.

Reply:
Good idea!